src/input/Keyboard.ts

```diff
@@ -17,13 +17,15 @@ keyToIntentMappings.set("ArrowDown", [ControllerIntent.PLAYER_DROP, ControllerIn
 keyToIntentMappings.set("ArrowLeft", [ControllerIntent.PLAYER_MOVE_LEFT, ControllerIntent.MENU_LEFT]);
 keyToIntentMappings.set("ArrowRight", [ControllerIntent.PLAYER_MOVE_RIGHT, ControllerIntent.MENU_RIGHT]);
 keyToIntentMappings.set("Enter", [ControllerIntent.PLAYER_INTERACT, ControllerIntent.CONFIRM]);
+keyToIntentMappings.set("NumpadEnter", [ControllerIntent.PLAYER_INTERACT, ControllerIntent.CONFIRM]);
 keyToIntentMappings.set("Escape", [ControllerIntent.ABORT, ControllerIntent.PAUSE]);
 keyToIntentMappings.set("ShiftLeft", [ControllerIntent.PLAYER_RUN]);
 keyToIntentMappings.set("ShiftRight", [ControllerIntent.PLAYER_RUN]);
 keyToIntentMappings.set("KeyE", [ControllerIntent.PLAYER_INTERACT, ControllerIntent.CONFIRM]);
 keyToIntentMappings.set("KeyF", [ControllerIntent.PLAYER_ACTION]);
 keyToIntentMappings.set("Digit1", [ControllerIntent.PLAYER_DANCE_1]);
-keyToIntentMappings.set("Digit2", [ControllerIntent.PLAYER_DANCE_2]);
```

Comment:
I think this line was removed accidentally.

Reply:
It was indeed. Has already been readded. 👍🏻